src/services/SiteInfoService.js

```diff
@@ -26,4 +28,11 @@ export const getSiteToShow = ({selectedSite, selectedPendingSite}) => {
     } else {
         return null;
     }
-};
\ No newline at end of file
+};
+
+export const returnImageForSiteKey = ({siteKey}) => {
+    return AsyncStorage.getItem(siteKey)
+        .then(siteImageData => {
+            return siteImageData;
+        });
```

Comment:
Added a service that we can call across the app. This particular function is used across the app to grab the image data for a site.